docs/content/developer/tutorials/sponsored-transactions/gas-station-server.mdx

```diff
@@ -1,11 +1,21 @@
 # The Gas Station Server
 
-:::info IOTA Gas Station
-
+:::info
 This guide is for developers who want to build their own gas station. If you'd rather just use sponsored transactions, IOTA provides a [Gas Station Component](../../../operator/gas-station/gas-station.mdx) that you can deploy in your environment.
-
 :::
 
+## Architecture and Flow Overview
+
+The workflow of the sponsored transaction is as follows:
+    - A user who does not have an IOTA balance wants to subscribe to the free trial option. They will send a request to the Gas Station server to sponsor their transaction alongside content type (Music, News or Movies).
+    - The [Gas Station server] will verify that this user has not used the free trial option before. If the user is eligible, the Gas Station will build a transaction (with gas payment from the sponsor's balance), sign it, and send back both the transaction and the signature separately as the response.
+    - The user, having received the transaction and the sponsor's signature, will sign the transaction and send it to the IOTA network. The user can do this using the CLI, Rust SDK or TS SDK. In this part, we will use the Rust SDK.
+    - The sponsor/admin (they can be different addresses, but for simplicity, we will assume they are the same) can now call the `publish` content function on the platform to publish the content to the user's address.
+
+![Architecture overview](../../../../site/static/img/developer/tutorials/sponsored-txs/sponsored-tx-arch.png)
```

Comment:
Why do the numbers have different sizes? Designers will kill us mister :trollface: ❤️